openlibrary/core/edits.py

```diff
@@ -90,6 +90,18 @@ def get_counts_by_mode(cls, mode='all', **kwargs):
             query = f'{query} WHERE {where_clause}'
         return oldb.query(query, vars=kwargs)[0]['count']
 
+    @classmethod
+    def get_submitters(cls):
+        oldb = db.get_db()
+        query = f'SELECT DISTINCT submitter FROM {cls.TABLENAME}'
+        return oldb.query(query)
```

Comment:
```suggestion
        return list(oldb.query(query))
```
Returning DB queries as a list is preferable to calling `CommunityEditsQueue.get_submitters().list()`.